config/routes.rb

```diff
@@ -1,3 +1,33 @@
+<<<<<<< HEAD
```

Comment:
Guys, it seems that you missed resolving the merge conflict here. Also, I saw another `routes.rb` file in `fakebook/config` folder, please keep just one copy of this file in the standard location.

Reply:
fixed